src/pages/ReimbursementAccount/BusinessInfo/substeps/IncorporationStateBusiness.tsx

```diff
@@ -1,39 +1,38 @@
 import React from 'react';
-import {withOnyx} from 'react-native-onyx';
+import {OnyxEntry, withOnyx} from 'react-native-onyx';
 import Form from '@components/Form';
 import StatePicker from '@components/StatePicker';
 import Text from '@components/Text';
 import useLocalize from '@hooks/useLocalize';
+import {SubStepProps} from '@hooks/useSubStep/types';
 import useThemeStyles from '@hooks/useThemeStyles';
 import * as ValidationUtils from '@libs/ValidationUtils';
-import {reimbursementAccountDefaultProps, reimbursementAccountPropTypes} from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
-import subStepPropTypes from '@pages/ReimbursementAccount/subStepPropTypes';
 import getDefaultValueForReimbursementAccountField from '@pages/ReimbursementAccount/utils/getDefaultValueForReimbursementAccountField';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import {ReimbursementAccount} from '@src/types/onyx';
+import * as OnyxCommon from '@src/types/onyx/OnyxCommon';
 
-const propTypes = {
-    /** Reimbursement account from ONYX */
-    reimbursementAccount: reimbursementAccountPropTypes,
+const companyIncorporationStateKey = CONST.BANK_ACCOUNT.BUSINESS_INFO_STEP.INPUT_KEY.INCORPORATION_STATE;
 
-    ...subStepPropTypes,
-};
+const validate = (values: OnyxCommon.Errors) => ValidationUtils.getFieldRequiredErrors(values, [companyIncorporationStateKey]);
 
-const defaultProps = {
-    reimbursementAccount: reimbursementAccountDefaultProps,
+type IncorporationStateBusinessOnyxProps = {
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
 };
 
-const companyIncorporationStateKey = CONST.BANK_ACCOUNT.BUSINESS_INFO_STEP.INPUT_KEY.INCORPORATION_STATE;
-
-const validate = (values) => ValidationUtils.getFieldRequiredErrors(values, [companyIncorporationStateKey]);
+type IncorporationStateBusinessProps = {
+    reimbursementAccount: ReimbursementAccount;
+} & SubStepProps;
```

Comment:
Please put types to the top of the file
```suggestion
type IncorporationStateBusinessProps = IncorporationStateBusinessOnyxProps & SubStepProps;
```